src/shared/components/ui/Button/Button.js

```diff
@@ -52,35 +49,46 @@ const Button = ({
   fitWidth,
   ['data-testid']: dataTestId
 }) => {
-  const elementClasses = cssClass({
-    'button--primary': variant === 'primary',
-    'button--secondary': variant === 'secondary',
-    'button--dropdown': variant === 'dropdown',
-    'button--transparent': variant === 'transparent',
-    'button--transparent-blue': variant === 'transparent-blue',
-    'button--dropdown-element': variant === 'dropdown-element',
-    'dropdown button--switcher': variant === 'switcher',
-    'button--tabs': variant === 'tabs',
-    'button--alert': variant === 'alert',
-    'button--icon': variant === 'icon',
-    'button--icon-transparent': variant === 'icon-transparent',
-    'button--white': variant === 'white',
-    'button--black': variant === 'black',
-    'button--action': variant === 'action',
-    'button--publish': variant === 'publish',
-    'button--mini': size === 'mini',
-    'button--tiny': size === 'tiny',
-    'button--small': size === 'small',
-    'button--medium': size === 'medium',
-    'button--large': size === 'large',
-    'button--input': size === 'input',
-    'button--center': align === 'center',
-    'button--left': align === 'left',
-    'button--right': align === 'right',
-    'button--svg': hasIcon === true,
-    'button--hide': hide === true,
-    'button--fit-width': fitWidth === true
-  })
+  const elementClassesButton = useStyles(
+    {
+      [styles.button]: true,
+      [styles['button--primary']]: variant === 'primary',
+      [styles['button--secondary']]: variant === 'secondary',
+      [styles['button--dropdown']]: variant === 'dropdown',
+      [styles['button--transparent']]: variant === 'transparent',
+      [styles['button--transparent-blue']]: variant === 'transparent-blue',
+      [styles['button--dropdown-element']]: variant === 'dropdown-element',
+      [styles['dropdown button--switcher']]: variant === 'switcher',
+      [styles['button--tabs']]: variant === 'tabs',
+      [styles['button--alert']]: variant === 'alert',
+      [styles['button--icon']]: variant === 'icon',
+      [styles['button--icon-transparent']]: variant === 'icon-transparent',
+      [styles['button--white']]: variant === 'white',
+      [styles['button--black']]: variant === 'black',
+      [styles['button--action']]: variant === 'action',
+      [styles['button--publish']]: variant === 'publish',
```

Comment:
to można zmniejszyć do jednej linii
`[styles[`button--${variant}`]]: variant`

to samo dotyczy
`[styles['button--right']]: align === 'right',` => `[styles[`button--${align}`]]: align`